WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsDetails.swift

```diff
@@ -90,13 +99,75 @@ private struct CustomFieldRow: View {
             Spacer()
         }
         .padding([.top, .bottom], Constants.hStackPadding)
+        .padding(.horizontal, Constants.hStackPadding)
+        .frame(minHeight: Constants.height)
+    }
+}
+
+private struct EditableCustomFieldRow: View {
+    /// Custom Field title
+    ///
+    let title: String
+
+    /// Custom Field content
+    ///
+    let content: String
+
+    /// Optional URL to link the content
+    ///
+    let contentURL: URL?
+
+    /// URL to display in `SafariSheet` in app
+    ///
+    @State private var displayedURL: URL?
+
+    /// Action to open URL with system handler
+    ///
+    @Environment(\.openURL) private var openURL
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading,
+                   spacing: Constants.spacing) {
+                Text(title)
+
+                if let url = contentURL { // Display content as a link if URL is provided
+                    Text(content)
+                        .font(.footnote)
+                        .foregroundColor(Color(.textLink))
+                        .safariSheet(url: $displayedURL)
+                        .onTapGesture {
+                            switch url.scheme {
+                            case "http", "https":
+                                displayedURL = url // Open in `SafariSheet` in app
+                            default:
+                                openURL(url) // Open in associated app for URL scheme
+                            }
+                        }
+                } else { // Display content as plain text
+                    Text(content)
+                        .footnoteStyle()
+                        .lineLimit(3)
+                }
+            }.padding([.leading, .trailing], Constants.vStackPadding)
+
+            Spacer()
+
+            // Chevron icon
+            Image(uiImage: .chevronImage)
+                .flipsForRightToLeftLayoutDirection(true)
+                .foregroundStyle(Color(.textTertiary))
+        }
+        .padding([.top, .bottom], Constants.hStackPadding)
+        .padding(.horizontal, Constants.hStackPadding)
```

Comment:
Same comment as above about padding.

Reply:
Updated as part of https://github.com/woocommerce/woocommerce-ios/pull/13639/commits/1706a3fa8a3a6a4e9abe47892321f5971b050da2